source/installation.rst

```diff
@@ -1,116 +1,128 @@
+         * Download the |percona-release| package
+           
+           .. code-block:: bash
 
-#. Install the |percona-release| utility.
-#. Set up the |Percona| repository for the required version of
-   |pdmdb|.
-#. Install the packages.
+              $ curl -O https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+         
+         * Install the package and dependancies
+           
+           .. code-block:: bash 
 
-|tip.run-all.root|
+              $ sudo apt install gnupg2 lsb-release ./percona-release_latest.generic_all.deb
 
-Install |percona-release|
-==============================================
+         * Refresh the local cache
 
-Follow the `installation instructions <https://www.percona.com/doc/percona-repo-config/installing.html>`_ relevant to your operating system in the Percona Software Repositories Documentation to install ``percona-release``.
+           .. code-block:: bash
 
-If you have previously installed |percona-release|, make sure it is `updated to the
-latest version <https://www.percona.com/doc/percona-repo-config/updating.html>`_.
+              $ supo apt update
 
-Set up |Percona| repository
-==============================================
+      #. Enable the repository
+         
+         Percona provides :ref:`two repositories <repo-overview>` for |pdmdb|. To enable a repo, we recommend using the ``setup`` command:
 
-As soon as |percona-release| is up-to-date, use the ``setup`` command to enable |Percona| repository for |pdmdb| (``pdmdb-4.4``). 
+         .. code-block:: bash
 
-.. code-block:: bash
+            $ sudo percona-release setup pdmdb-4.4
 
-   $ sudo percona-release setup pdmdb-4.4
+      #. Install |pdmdb| packages:
+         
+         .. code-block:: bash
 
-.. hint::
+            $ sudo apt install percona-server-mongodb percona-backup-mongodb
 
-   To enable a minor version repository (e.g. pdmdb-4.4.0), use the following command:
+   .. tab:: On RHEL / derivatives
 
-   .. code-block:: bash
-   
-      $ sudo percona-release setup pdmdb-4.4.0
+      |tip.run-all.root|
 
-.. _pdmdb-install:
-   
-Install packages
-==============================================
+      1. Install |percona-release|
+         
+         Install the |percona-release| repository management tool to subscribe to Percona repositories:
+         
+         .. code-block:: bash
 
-Install |pdmdb| components using the package manager of your operating system.
+            $ sudo yum install -y https://repo.percona.com/yum/percona-release-latest.noarch.rpm
+         
+      #. Enable the repository 
+         
+         Percona provides :ref:`two repositories <repo-overview>` for |pdmdb|. To enable a repo, we recommend using the ``setup`` command:
 
-Install on *Debian / Ubuntu*
-----------------------------------------------
+         .. code-block:: bash
 
-Use the following commands to install |PSMDB| and / or
-|PBM|:
+            $ sudo percona-release setup pdmdb-4.4
```

Comment:
Do we have a page somewhere describing what other versions do we support and how to set it up?

Like pdmdb-5.0 I would assume is the Distribution for 5.0.

Reply:
These are in the corresponding version's docs.